cli/src/cli_util.rs

```diff
@@ -338,6 +341,14 @@ impl CommandHelper {
         load_template_aliases(ui, &self.data.layered_configs)
     }
 
+    /// Loads fileset aliases from the configs.
+    ///
+    /// For most commands that depend on a loaded repo, you should use
+    /// `WorkspaceCommandHelper::fileset_aliases_map()` instead.
+    fn load_fileset_aliases(&self, ui: &Ui) -> Result<FilesetAliasesMap, CommandError> {
+        load_fileset_aliases(ui, &self.data.layered_configs)
+    }
```

Comment:
nit: just inline this because it's unlikely we'll need to parse file paths without loading a workspace.